test/regress/run_regression.py

```diff
@@ -382,6 +382,15 @@ def run_regression(unsat_cores, proofs, dump, use_skip_return_code,
         if extra_command_line_args:
             command_line_args_configs.append(all_args +
                                              extra_command_line_args)
+            
+        extra_command_line_args_proof = []
+        if proofs and re.search(r'^(unsat|valid)$', expected_output):
+            if '--no-check-proofs' not in all_args and \
+               '--check-proofs' not in all_args:
```

Comment:
IIRC, we were combining proofs and unsat cores in the same call. This change will call the solver three times instead of two (at least on the unsat benchmarks). Are we ok with this?

Reply:
Yes, this was the intention.  In the future, unsat cores and proofs will be fairly disjoint (apart from preprocessing).